Chinese/the-right-way/to_be_a_coder.md

```diff
@@ -0,0 +1,7 @@
+悟空在天庭当斗战胜佛当了很多年，这个体制内的差事一直稳定有油水，然而近些年的反贪腐运
+动轰轰烈烈地开展着，日子渐渐变得不好过起来。
+
+悟空听人说二十一世纪是生物的时代，但又看到互联网的风口中八戒都能飞起来，一时拿不定主
+意是转行进入
+[生物行业]()，
+[还是成为程序猿]()。
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.